.github/workflows/buildRelease.yml

```diff
@@ -115,7 +115,7 @@ jobs:
         cd $GITHUB_WORKSPACE
         TM_PACKAGE_VERSION=${{ github.event.inputs.python_package_version }}
         printf "TORCH_MLIR_PYTHON_PACKAGE_VERSION=%s\n" $TM_PACKAGE_VERSION > ./torch_mlir_package_version
-        TM_PYTHON_VERSIONS=${{ matrix.py_version }} TM_PACKAGES=${{ matrix.package }} ./build_tools/python_deploy/build_linux_packages.sh
+        TM_PYTHON_VERSIONS=${{ matrix.py_version }} TM_PACKAGES=${{ matrix.package }} TM_TORCH_VERSION="stable" $env:TORCH_MLIR_ENABLE_LTC='0' ./build_tools/python_deploy/build_linux_packages.sh
```

Comment:
Hi @ashay, after going through the `build_linux_packages.sh`, I found out that the LTC build is disabled for the stable Pytorch for both in-tree and out-of-tree build. I don't understand why is it failing during the release package build since it should be disabled when we set the torch package version to stable. See: https://github.com/llvm/torch-mlir/blob/main/build_tools/python_deploy/build_linux_packages.sh#L220-L222

Reply:
Ah, I see your point now.  In the release builds, we build the wheel file using setup.py which uses the `TORCH_MLIR_ENABLE_LTC` environment variable, whereas the non-release (i.e. CI) builds decide whether to enable LTC based on `TM_TORCH_VERSION`.  I'll submit a patch to make this consistent in the coming days.

My apologies for the noise, since your patch is mostly correct.  We should drop the `TM_TORCH_VERSION=stable` part and drop the `$env:` prefix for `TM_TORCH_VERSION`, but other than that, we should be good to go.